cns/restserver/util.go

```diff
@@ -773,13 +802,13 @@ func (service *HTTPRestService) populateIPConfigInfoUntransacted(ipConfigStatus
 	return nil
 }
 
-// isNCWaitingForUpdate :- Determine whether NC version on NMA matches programmed version
+// isNCWaitingForUpdateV2 :- Determine whether NC version on NMA matches programmed version
 // Return error and waitingForUpdate as true only CNS gets response from NMAgent indicating
 // the VFP programming is pending
 // This returns success / waitingForUpdate as false in all other cases.
-func (service *HTTPRestService) isNCWaitingForUpdate(
-	ncVersion, ncid string,
-) (waitingForUpdate bool, returnCode types.ResponseCode, message string) {
+// V2 is using the nmagent get nc version list api v2 which doesn't need authentication token
+func (service *HTTPRestService) isNCWaitingForUpdateV2(
```

Comment:
If we're not keeping both versions of this function in code, I'd suggest to just call this `isNCWaitingForUpdate` (the existing name) and just update the implementation.

Reply:
OK